deprecated.go

```diff
@@ -1,8 +1,9 @@
 package gin
 
 import (
-	"github.com/gin-gonic/gin/binding"
 	"net/http"
+
+	"github.com/Zaibon/gin/binding"
```

Comment:
make sense, and does look neater.  was just curious as to why this change was coming together with the actual flow change